CalibTracker/SiPixelTools/plugins/SiPixelErrorsDigisToCalibDigis.cc

```diff
@@ -129,22 +129,21 @@ void SiPixelErrorsDigisToCalibDigis::endJob() {
 
 // ------------ helper functions ---------------------------------------------------------
 
-MonitorElement* SiPixelErrorsDigisToCalibDigis::bookDQMHistogram2D(uint32_t detid,
-                                                                   std::string name,
-                                                                   std::string title,
-                                                                   int nchX,
-                                                                   double lowX,
-                                                                   double highX,
-                                                                   int nchY,
-                                                                   double lowY,
-                                                                   double highY) {
+SiPixelErrorsDigisToCalibDigis::MonitorElement* SiPixelErrorsDigisToCalibDigis::bookDQMHistogram2D(uint32_t detid,
```

Comment:
I’m fairly certain you could have used ```auto``` as the return type.

Reply:
Probably yes, but that requires changes in two places (header and implementation). Also the return type might not (always) be obvious, so I went the most obvious route.